src/compute/src/render/mod.rs

```diff
@@ -656,6 +660,22 @@ where
                 if let Some(token) = &self.shutdown_token.get_inner() {
                     oks = oks.with_token(Weak::clone(token));
                 }
+
+                if let Some(max_iter) = max_iter {
+                    // We swallow the results of the `max_iter`th iteration, because
```

Comment:
Interesting block to test

Reply:
Hmm, well, this is definitely covered in `with_mutually_recursive.slt` (by most of the new tests), as this is the core of the PR. I verified this now by putting a crash into this block and running `with_mutually_recursive.slt`.

What tests does coverage take into account? Does it consider slt tests? Or how can it be that it shows it as not covered?

Reply:
It should consider slt tests: https://github.com/MaterializeInc/materialize/blob/main/doc/developer/code-coverage.md I'm seeing a problem with the coverage data here, will investigate.